docs/source/deploy.rst

```diff
@@ -0,0 +1,151 @@
+======
+Deploy
+======
+
+The deployment docs are split up in two parts. First there is the
+general section, which should always be followed. Then there is a cloud
+service provider specific section of which one provider should be chosen.
+
+If you are not sure where to deploy your app, we suggest Heroku. You can test 
+deploying and serving your app without having to enter any credit card details,
+ and with very little prior experience of deployments.
```

Comment:
This line is giving an `WARNING: Unexpected indentation` error when trying locally.

```suggestion
and with very little prior experience of deployments.
```

Reply:
fixed. I dont' use a linter for the docs. What linter do you use? Maybe it would be a good idea to add it to the doc-requirements?